mw-webapp/src/pages/Page404/Page404.module.scss

```diff
@@ -0,0 +1,10 @@
+@import "src/styles/variables.scss";
+
+.container {
+  display: flex;
+  width: 100%;
+  height: 70vh;
```

Comment:
why height of page's container is 70vh?

Reply:
If you take the entire height of the screen, scrolling appears.

Reply:
page should have dynamic height depends on height of content